test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -46,6 +46,7 @@
 import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableSet;
 
+import org.apache.cassandra.auth.AuthCacheService;
 import org.apache.cassandra.exceptions.InvalidRequestException;
 import org.apache.cassandra.io.util.File;
```

Comment:
super nit: These three imports should move down with the others of their kind :)